.github/workflows/build-rtools40.yml

```diff
@@ -1,27 +1,6 @@
 name: rtools40
 on:
-  push:
-    branches:
-      - dev
-      - release-*
-      - refs/tags/*
-    paths-ignore:
-      - '.github/workflows/quarto-render.yml'
-      - '_quarto.yml'
-      - 'quarto-materials/*'
-      - '**/.md'
-      - 'doc/source/conf.py'
-      - 'tiledb/sm/c_api/tiledb_version.h'
-  pull_request:
-    branches:
-      - '*'  # must quote since "*" is a YAML reserved character; we want a string
-    paths-ignore:
-      - '.github/workflows/quarto-render.yml'
-      - '_quarto.yml'
-      - 'quarto-materials/*'
-      - '**/.md'
-      - 'doc/source/conf.py'
-      - 'tiledb/sm/c_api/tiledb_version.h'
+  workflow_call:
```

Comment:
Should this be an empty block?

Reply:
Yes, this trigger section is necessary to chain workflows (https://docs.github.com/en/actions/using-workflows/reusing-workflows#creating-a-reusable-workflow).